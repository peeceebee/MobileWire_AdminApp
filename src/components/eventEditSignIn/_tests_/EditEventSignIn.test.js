import EditEventSignIn from "../EventEditSignIn";
import React from "react";
import { shallow } from "enzyme";

describe("EditEventSignIn", () => {
  let wrapper = shallow(<EditEventSignIn />);
  it("Should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
