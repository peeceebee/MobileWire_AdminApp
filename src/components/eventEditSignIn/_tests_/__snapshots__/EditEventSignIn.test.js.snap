// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditEventSignIn Should match the snapshot 1`] = `ShallowWrapper {}`;
